src/test/js/edgeCases/folder.js

```diff
@@ -204,4 +204,31 @@ module.exports = {
             blueRunDetailPage.assertBasicLayoutOkay();
         });
     },
+    /**
+     * test open blueocean from classic - a normal folder page in classic jenkins.
+     * <p>
+     * It should send the user to the top level blue ocean page (pipelines).
+     * @param browser
+     */
+    'step 11': function(browser) {
+        var classicGeneral = browser.page.classicGeneral();
+
+        // Go to a folder along the path to the MBP, but one
+        // of the parent folders i.e. not the MBP project folder.
+        classicGeneral.navigateToRun('job/anotherFolder/job/三百/job/ñba');
```

Comment:
We could, but not needed imo i.e. the main thing is that it's **not** the MBP folder at the end of the path.